### PR TITLE
65 rollup with empty select fix

### PR DESCRIPTION
**Pull request by johguentner**
[fix: check if content is set for rollup properties](https://github.com/5am-code/laravel-notion-api/commit/2bfcee476544ea594ef84040e72135bb0c91d8a3) 

- specific rollup properties can be null, which lead to issues of setting the content of the property
- this fix checks the existence of content and skips setting the content, if it is null (e.g. empty selects, multi_selects etc.)
- fixes issue: Rollup that refers to an empty Select property throws exception during database query

### CODE REVIEW

**Comment by johguentner** [comment]
Hey @mechelon,
tests for this edge case are added. ✅ 🎉 
If your review passes, let's add this to v0.8.0.

**Comment by johguentner** [comment]
Hey @mechelon,
tests are modified (due to breaking changes) and additional ones added. ✅ 🎉
If your review passes, let's add this to v0.8.0 as well.

File: tests/EndpointDatabaseTest.php [hunk 1]
```diff
@@ -157,4 +157,32 @@ public function it_throws_a_notion_exception_bad_request()
 
         Notion::database('8284f3ff77e24d4a939d19459e4d6bdc')->query();
     }
+
+    ## EDGE CASES
```
Comment (mechelon) [inline tests/EndpointDatabaseTest.php]: I would remove this comment, it's still a regular test I think 🤷‍♀️

**Review by mechelon — APPROVED** [review]
